Utils/chessCommands.py

```
@@ -48,6 +48,6 @@ def getRandomOpeningSpecific(name, side=None):
 
 
 def getRandom960():  # Get random 960 position.
-    lines = open('chess960 FENS.txt').read().splitlines()
+    lines = open('../Data/chess960 FENS.txt').read().splitlines()
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)